lib/matplotlib/backends/_backend_pdf_ps.py

```diff
@@ -24,28 +25,59 @@ def get_glyphs_subset(fontfile, characters):
     Subset a TTF font
 
     Reads the named fontfile and restricts the font to the characters.
-    Returns a serialization of the subset font as file-like object.
+    Returns a TTFont object.
 
     Parameters
     ----------
     symbol : str
         Path to the font file
     characters : str
         Continuous set of characters to include in subset
+
+    Returns
+    -------
+    fontTools.ttLib.ttFont.TTFont
```

Comment:
Long term, it could make sense to link to the documentation? (Although fontTools are not in the intersphinx_mapping yet.)